public/locales/en-US/translations.json

```diff
@@ -148,6 +150,22 @@
   "transaction_EnableAmendment": "Enable Amendment",
   "transaction_SetFee": "Set Fee",
   "transaction_UNLModify": "UNL Modify",
+  "ledger_entry_type_AccountRoot":"Account Root",
+  "ledger_entry_type_Amendments":"Amendments",
+  "ledger_entry_type_Check":"Check",
+  "ledger_entry_type_DepositPreauth":"Deposit Preauth",
+  "ledger_entry_type_DirectoryNode":"Directory Node",
+  "ledger_entry_type_Escrow":"Escrow",
+  "ledger_entry_type_FeeSettings":"Fee Settings",
+  "ledger_entry_type_LedgerHashes":"Ledger Hashes",
+  "ledger_entry_type_NegativeUNL":"Negative UNL",
+  "ledger_entry_type_NFTokenOffer":"NFToken Offer",
+  "ledger_entry_type_NFTokenPage":"NFToken Page",
+  "ledger_entry_type_Offer":"Offer",
+  "ledger_entry_type_PayChannel":"Pay Channel",
+  "ledger_entry_type_RippleState":"Ripple State",
+  "ledger_entry_type_SignerList":"Signer List",
```

Comment:
Can add more meaningful UI names for better understanding. 
Reference: https://xrpl.org/ledger-object-types.html